python/ml4ir/applications/ranking/tests/data/configs/feature_config.yaml

```diff
@@ -62,6 +62,19 @@ features:
     feature_layer_info:
       type: numeric
       shape: null
+      fn: tf_native_op
+      args:
+        ops:
+          - fn: tf.math.add
+            kwargs:
+              y: 0.
+          - fn: tf.math.subtract  # The goal here is to see the end-to-end functionality of tf_native_op without modifying the tests
```

Comment:
so if we add then subtract 0, do we notice?

also, clip_by_value between 0 and 1M?  will our unit tests ever see data outside that range in the first place?

Reply:
No, my goal was to mostly see if this would lead to an error. I commented above, this does not do anything to change the feature values or the resulting scores. I did this because I didn't want to modify all the other gold values from other tests. But I can do that, just to be safe.